src/main/java/calculator/domain/Operator.java

```diff
@@ -0,0 +1,23 @@
+package calculator.domain;
+
+public enum Operator {
+    PLUS("+"),
```

Comment:
```suggestion
    PLUS("+") {
        @Override
        public int calculate(int operand1, int operand2) {
            return operand1 + operand2;
        }
    }
```
enum으로 구현했으니 각 enum에 해당하는 계산 로직을 위와 같이 구현한 후 반복해서 발생하고 있는 if문을 제거해 보는 시도를 해보면 어떨까?
힌트: enum 또한 class와 완벽히 같은 방식으로 인터페이스, 상속 등의 구현이 가능함